Makefile

```diff
@@ -68,6 +68,23 @@ release: manifests
 	mkdir -p release
 	kustomize build config/default > release/actions-runner-controller.yaml
 
+	# build helm charts	
+	# copy controller manifest to helm templates
+	cp -v release/actions-runner-controller.yaml charts/actions-runner-controller/templates
```

Comment:
Consider trying to use `split -p '---'` to split up all the manifests if possible (though no idea how you'd give each file sensible names without having to use `yq` to extract the `metadata.name` and `kind`).

More importantly though, the CRDs should get `cp`ed into the `crds` folder of the chart rather than the templates: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you